updater/updater.go

```diff
@@ -133,6 +135,9 @@ func Run(config *config.UpdaterConfig, datastore database.Datastore, st *utils.S
 					nextUpdate = lockExpiration
 				}
 			}
+		} else {
+			DbFinishUpdate = false
```

Comment:
This is unfortunately far from enough to determine whether the initial update succeeded or not. First of all, your else block will be executed when it is not yet time to update and set the update status as unfinished, this does not really make sense. Also,
- Consider the case in which you have multiple Clair instances running and only one of them realizes the updates.
- Consider the case in which Clair is restarted.